<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="databasesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>129</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Database selected:</string>
    </property>
   </widget>
   <widget class="QListView" name="groups">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="subjects">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="semesters">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Semester 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semester 10</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="databaseSelected">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListView" name="students">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>311</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="controlType">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="groupAddButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subjectAddButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Subject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlAddButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentAddButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groupDeleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subjectDeleteButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Subject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlDeleteButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentDeleteButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Student</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="controlAddDate">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="controlAddTime">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>590</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentSubjectNameLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentControlTypeLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentStudentFullNameLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentStudentNumberLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="currentControlDate">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="currentControlTime">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="currentControlResult">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зачёт</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Незачёт</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="controlResultLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currentResultSave">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>270</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currentResultDelete">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Result</string>
    </property>
   </widget>
   <widget class="QListView" name="controlAddTeacher">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="controlAddType">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Экзамен</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Доп. экзамен</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зачёт</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="subjectAddName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentGroupLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>52</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Группа:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentGroupNumber">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSession_v0_1a">
    <property name="title">
     <string>Session v0.2a</string>
    </property>
   </widget>
   <addaction name="menuSession_v0_1a"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
